os/debian/Dockerfile.debian

```diff
@@ -27,6 +32,10 @@ ARG PYTHON_VERSION=3.10.8
 # Recommended: LC_ALL=C.UTF-8
 ENV LC_ALL=C.UTF-8
 
+# Multi platform support - override this arg with `arm64` string to build for Mac m1 for example
+ARG ARCH=amd64
+ENV ARCH=${ARCH}
+
```

Comment:
seems this is a dupe of line 15
```suggestion
```

Reply:
nope, different image. https://github.com/cloudposse/geodesic/blob/4851de6c2419149a0c3500955816956c3e17c9ee/os/debian/Dockerfile.debian#L27 did you have a chance to make changes locally and build it?

Reply:
I think `TARGETARCH` will straight the code and some of this won't be necessary. I think I'll go with `BUILDARCH`, tho, as it's `TARGETARCH` appears to be available only when using `--platform` parameter so the latter one seems more stable. Correct me if I'm wrong

Reply:
Oops I didn't read the whole file, only the snippets in the diff, didn't realize it is a multi-stage build

IIRC TARGETARCH and BUILDARCH are both available, and BUILDARCH is the arch of the host machine so it won't work for doing a multi-platform build in CI

But again this is from memory 😝 I will test today and let you know

Reply:
TARGETARCH should be set on all buildkit-based builds which is default in recent Docker but I'm seeing it must be placed after the FROM. I will make a PR to your fork with a working build.